scripts/config/templates/nginx.conf.erb

```diff
@@ -33,16 +33,20 @@ http {
     * 2;
   }
 
-  # TODO update to use remote_addr instead of request_id
-  split_clients "${request_id}_SPEEDY_SEARCH" $speedy_search_assignment {
+  split_clients "${request_id}_SPEEDY_SEARCH_FF" $speedy_search_ff {
```

Comment:
We should use `remote_address` instead. Otherwise this will switch back and forth all the time.